client/components/Item/RelatedItems.vue

```diff
@@ -109,12 +110,25 @@ export default Vue.extend({
     async refreshItems(): Promise<void> {
       this.loading = true;
 
+      let excludeArtistsIds = [];
+
+      if (this.item.AlbumArtists) {
+        excludeArtistsIds = this.item.AlbumArtists.map(
+          (albumArtist: { Id: string }) => albumArtist.Id
+        );
+      }
+
+      const requestParameters: LibraryApiGetSimilarItemsRequest = {
```

Comment:
This could be directly inside `getSimilarItems` like it was before.

Currently, it's also outside of the condition on `this.item.Id`, which kind of negates the purpose of that condition and could lead to issues.